python_ta/__init__.py

```diff
@@ -101,7 +100,8 @@ def _check(module_name='', reporter=ColorReporter, number_of_messages=5, level='
                                 # TODO: Fix this test
                                 #'python_ta/checkers/invalid_range_index_checker',
                                 'python_ta/checkers/assigning_to_self_checker',
-                                'python_ta/checkers/always_returning_checker'])
+                                'python_ta/checkers/always_returning_checker',
+                                'python_ta/checkers/pycodestyle_checker'])
```

Comment:
For now, please continue to use the `pep8` flag so that this checker is still disabled by default.